<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>superPage2Edit</class>
 <widget class="QWidget" name="superPage2Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(220, 220, 220);
}
QLabel#lab_Ico,QLabel#lab_Title{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
}

QLineEdit {
	border: 1px solid #B8B8B8;
	border-radius: 5px;
	padding: 2px;
	background: none;
	selection-background-color: #454648;
}

QLineEdit[echoMode=&quot;2&quot;] { 
	lineedit-password-character: 9679; 
}

.QGroupBox{
	border: 1px solid #B8B8B8;
	border-radius: 5px;
}

/*提示标题*/
.QToolTip
{
    background: #f6f5ec;        /*背景颜色*/
    opacity:255;                /*透明度*/
    border: 1px solid #130c0e;  /*边框宽度样式及颜色*/
    color:#130c0e;              /*字体颜色*/
}
.QPushButton{
	border-style: none;
	border: 0px;
	color: #F0F0F0;
	padding: 5px;	
	border-radius:5px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

.QPushButton[focusPolicy=&quot;0&quot;] {
	border-style: none;
	border: 0px;
	color: #F0F0F0;
	padding: 0px;	
	min-height: 10px;
	border-radius:3px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

.QPushButton:hover{ 
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #B8B8B8, stop:1 #D6D6D6);
}

.QPushButton:pressed{ 
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A);
}

QPushButton#btnMenu,QPushButton#btnMenu_Min,QPushButton#btnMenu_Max,QPushButton#btnMenu_Close{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
}

QPushButton#btnMenu:hover,QPushButton#btnMenu_Min:hover,QPushButton#btnMenu_Max:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(25, 134, 199, 0), stop:1 #B8B8B8);
}

QPushButton#btnMenu_Close:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(238, 0, 0, 128), stop:1 rgba(238, 44, 44, 255));
}

QCheckBox {
	spacing: 2px; 
}

QCheckBox::indicator {
	width: 20px;
	height: 20px;
}

QCheckBox::indicator:unchecked {
	image: url(:/qss/clientcss/checkbox_unchecked.png);
}

QCheckBox::indicator:checked {
	image: url(:/qss/clientcss/checkbox_checked.png); 
}

QRadioButton {
	spacing: 2px;
}

QRadioButton::indicator {
	width: 15px; 
	height: 15px; 
}

QRadioButton::indicator::unchecked {
	image: url(:/qss/clientcss/radio_normal.png); 
}

QRadioButton::indicator::checked {
	image: url(:/qss/clientcss/radio_selected.png); 
}

QComboBox,QDateEdit{
	border-radius: 3px;
	padding: 1px 10px 1px 5px;
	border: 1px solid #B8B8B8;
}

QComboBox::drop-down,QDateEdit::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px; 
	border-left-width: 1px;
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
	border-left-color: #B8B8B8;
}

QComboBox::down-arrow,QDateEdit::down-arrow {
	image: url(:/qss/clientcss/array_down.png); 
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>员工信息修改</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>53</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>工号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="workid_lineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>入职时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="datetime_lineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>49</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>年龄：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="agelineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>身份证号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="idcard_lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>联系电话：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="phone_lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,4">
       <property name="leftMargin">
        <number>29</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>住址：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="address_lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QPushButton" name="update_pushButton">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_pushButton">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
